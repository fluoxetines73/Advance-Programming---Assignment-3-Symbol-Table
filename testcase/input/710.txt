PRINT
PRINT
INSERT f_FF number
BEGIN
ASSIGN _@_$$d 123
PRINT
BEGIN
INSERT d number
LOOKUP f_FF
RPRINT
END
INSERT f_FF number
UWAAAA
BEGIN
PRINT
END
END