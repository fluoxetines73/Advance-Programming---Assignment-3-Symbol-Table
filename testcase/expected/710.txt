Invalid: ASSIGN _@_$$d 123